salt/modules/smartos_nictagadm.py

```diff
@@ -163,10 +163,7 @@ def add(name, mac, mtu=1500):
             return {'Error': '{0} is not present on this system.'.format(mac)}
 
     if mac == 'etherstub':
-        cmd = 'nictagadm add -l -p mtu={mtu} {name}'.format(
-            mtu=mtu,
-            name=name
-        )
+        cmd = 'nictagadm add -l {0}'.format(name)
```

Comment:
Can we keep this consistent within the file? and use ether ```{name}``` here too or correct the other ones to use this unnamed format.

Reply:
Thanks, I've chosen to use the unnamed format as that appears to have been more common in the rest of the file.